03/app.js

```diff
@@ -1,5 +1,33 @@
 document.addEventListener('DOMContentLoaded', init);
 
-function init() {
-    console.log('DOM');
-}
\ No newline at end of file
+async function init() {
+	console.log('DOM');
+	button();
+}
+
+async function button() {
```

Comment:
Funkcja to działanie - czyli czynność (chyba, że to konstruktor, ale to inny temat) dlatego powinna być czasownikiem.